dev/CommonStyles/CommonStyles.vcxitems

```diff
@@ -111,11 +111,29 @@
       <Version>RS1</Version>
       <Type>ThemeResources</Type>
     </Page>
+    <Page Include="$(MSBuildThisFileDirectory)GridViewItem_themeresources.xaml">
+      <UseVisualStyle>LatestWithoutVisualUpdate</UseVisualStyle>
+      <Version>RS1</Version>
+      <Type>ThemeResources</Type>
+      <UseNonstandardCondtionalXAML>true</UseNonstandardCondtionalXAML>
+    </Page>
+    <Page Include="$(MSBuildThisFileDirectory)GridViewItem_themeresources_vu.xaml">
```

Comment:
the name rule is GridViewItem_themeresources_rs2.xaml, GridViewItem_themeresources_21H1.xaml